### ISSUE TITLE
ctpu not working when username has an underscore character

### ISSUE THREAD

**Issue by droidicus** [tensorflow/tpu#90]
When attempting to allocate a new flock with ctpu such as ```ctpu up```, and with a username with an underscore character (as in first_last) ctpu fails with the following error:
```
2018/03/13 14:35:08 googleapi: Error 400: Invalid value for field 'resource.name': 'first_last'. Must be a match of regex '(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)', invalid
```

This is avoidable by using ```ctpu -name=first-last up```, however it would be nice if this was handled more elegantly by ctpu.

**Comment by saeta** [comment]
Ack! I have a patch in review that I'll work on rolling into the next ctpu release. Thanks for filing this bug and for using ctpu!

**Comment by droidicus** [comment]
Sure thing, thanks for looking at this.

**Comment by saeta** [comment]
Hey @droidicus I just wanted to follow up. I've released `ctpu` version `0.6`. Please let me know if you the new behavior is closer to what you expect. Thanks for trying out `ctpu`!

**Comment by droidicus** [comment]
I am not sure if this was what was intended, but I now cannot do anything without specifying the ```name``` flag, even attempting to query the version is giving an error:
```
$ ctpu version
Error encountered while creating configuration: flock name "first_last" is not a valid flock name (must match regex: "^[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?$")
```

**Comment by saeta** [comment]
Hey @droidicus and @revilokeb ,

Sorry about the -name issues. I've released ctpu 0.7 (which should be live now) that should make it so you don't always need to specify -name for all the commands. Let me know if you still need to specify --name for any/all ctpu commands!

Thanks!
-Brennan
